persist/fs/manager.go

```diff
@@ -23,6 +25,19 @@ const (
 	persistManagerPersisting
 )
 
+type persistManagerMetrics struct {
```

Comment:
Hey I think recording latency within the persist manager might be too fined grained, it might make sense to first record the `Flush` latency at the namespace level to start with, and if we see latency issues, then we can refine the details.

Speaking of which, we should also have a gauge metric that's periodically reported that tells us when it's flushing and when it's not.